config/initializers/monkeypatches/nonexistent_exhibit_patch.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Rails.application.config.to_prepare do
```

Comment:
We should add a link to the upstream code for future reference. I'll go ahead and commit it to this PR.